cmd/config_set.go

```diff
@@ -71,6 +71,10 @@ you want to change. Other options will remain unchanged.`,
 		} else {
 			auther, err = d.store.Auth.Get(set.AuthMethod)
 			checkErr(err)
+			// check if there are new flags for existing auth method
+			set.AuthMethod, auther = getAuthentication(flags, set, auther)
```

Comment:
Perhaps we could simplify this and merge with line 68, passing `nil` if `hasAuth == false`.

Reply:
this is done